doc/source/MANUAL.rst

```diff
@@ -354,6 +354,42 @@ There is also an option on ``stestr run``, ``--random``/``-r`` to randomize the
 order of tests as they are passed to the workers. This is useful in certain
 use cases, especially when you want to test isolation between test cases.
 
+
+User Config Files
+-----------------
+
+If you prefer to have a different default output or setting for a particular
+command stestr enables you to write a user config file to overide the defaults
+for some options on some commands. By default stestr will look for this config
+file in ``~/.stestr.conf`` and ``~/.config/stestr.conf`` in that order. You
```

Comment:
I feel this is a bit confusing naming. I was thinking the format would be "INI" which is same as ``.stestr.conf``. But this is a yaml file. Why not use .stestr.yaml/stestr.yaml here?

Reply:
I can change it back to yaml, I don't have a strong preference one way or the other.